src/raw.js

```diff
@@ -136,6 +140,10 @@ function replaceKeyBindings(raw) {
     return full.replace(key, '?')
   })
 
+  emptyBindings.map(function(binding) {
+    sql = sql.replace(new RegExp(` ${binding}|${binding} |${binding}`, 'g'), '');
```

Comment:
@elhigu It sounds reasonable for most use cases, such as named bindings for setting or inserting values, but when you start using the named bindings for columns in tablenames it might seem odd to replace it with `DEFAULT`, or really anything at all.

```javascript
knex.raw('SELECT * FROM table WHERE :column: = :value', {column: 'users.name', value: 'knex'}).toString();
///... WHERE "users"."name" = 'knex'
```

```javascript
knex.raw('SELECT * FROM table WHERE :column: = :value', {column: void 0, value: 'knex'}).toString();
//... WHERE DEFAULT = 'knex'
```

I do however agree that completely erasing the binding from the query which we did previously, and currently still are, is very obnoxious and should not be done.

Perhaps if it's a regular binding such as `:val` we can use the `DEFAULT` approach, but if it's `:column:` we leave it the way it is? I don't know.. Can't come up with something that is consistent throughout..